server/thirdparty/integrationConfig.js

```diff
@@ -44,6 +44,24 @@ const IntegrationConfig = {
       },
     }],
   },
+  Github: {
+    icon_path: 'slackIcon.png',
+    Events: [{
+      EventName: 'Create New Issue',
+      EventType: 'Action',
+      OutputsWeGet: {
```

Comment:
We don't get outputs from an action

Reply:
This template is not used at all. This can be completely ignored.